src/schrodinger/sketcher/molviewer/view.h

```diff
@@ -36,10 +36,16 @@ class SKETCHER_API View : public QGraphicsView
 
   public slots:
     /**
-     * Scale and center the view matrix so that all objects in the scene are
+     * Scale and center the view matrix so that objects in the scene are
      * visible, but avoid zooming in too much for small molecules
+     * @param selection_only If true, only consider selected objects
      */
-    void fitToScreen();
+    void fitToScreen(bool selection_only = false);
+
+    void fitAllToScreen()
+    {
+        fitToScreen(false);
+    }
```

Comment:
Do we need this method if fitToScreen's default value for `selection_only` is `false`?  You should be able to connect `MolModel::newMoleculeAdded` directly to `fitToScreen`.

Reply:
yeah that's what I thought too! but apparently default values don't work that way for connect, and I had to add this function.